client/blocks/signup-form/social.jsx

```diff
@@ -75,7 +84,14 @@ class SocialSignupForm extends Component {
 						responseHandler={ this.handleGoogleResponse }
 						redirectUri={ redirectUri }
 						uxMode={ uxMode }
-						onClick={ this.trackGoogleLogin }
+						onClick={ this.trackSocialLogin.bind( null, 'google' ) }
```

Comment:
Binding just the argument and not the `this` feels a little unnatural to me.

What do you think of this?

```suggestion
						onClick={ () => this.trackSocialLogin( 'google' ) }
```

More symbols but still a one-liner :)

Reply:
I prefer that suggestion as well.